### ISSUE TITLE
translation errors for devise and devise_invitable only displaying en.devise.failure.unauthenticated

### ISSUE THREAD

**Issue by thuy-econsys** [thuy-econsys/rails_app#2]
Per instructions from Devise Wiki [How To: Require admin to activate account before sign_in - Model changes](https://github.com/heartcombo/devise/wiki/How-To:-Require-admin-to-activate-account-before-sign_in#model-changes) `config/locales/devise.en.yml` was updated to include:
```yml
en:
  devise:
    registrations:
      user:
        signed_up_but_not_approved: 'You have signed up successfully but your account has not been approved by your administrator yet'
```

User model was updated to override **Devise::Models::Authenticatable** methods: 
```ruby
  def active_for_authentication? 
    super && approved? 
  end 
  
  def inactive_message 
    approved? ? super : :not_approved
  end
```

Initially there was an alert `translation missing: en.devise.failure.user.not_approved` so `config/locales/devise.en.yml` was updated to include:
```yml
en:
  devise:
      failure:
        user:
          not_approved: "Your account has not been approved by your administrator yet."
```

...but now changed and passing test for login:
```yaml
en:
  devise:
      failure:
        not_approved: "Your account has not been approved by your administrator yet."
```

Alerts displaying now are still only displaying `You need to sign in or sign up before continuing.` for signup:
```yml
en:
  devise:
     failure:
       unauthenticated: "You need to sign in or sign up before continuing."
```

Possible quirk having to do with also using `letter_opener` gem ¯\\_(ツ)_/¯

**Comment by thuy-econsys** [comment]
https://github.com/heartcombo/devise/wiki/How-To:--Redirect-with-locale-after-authentication-failure

**Comment by thuy-econsys** [comment]
[link](https://github.com/heartcombo/devise/wiki/How-To:-Require-admin-to-activate-account-before-sign_in#reset-password-instructions) has: 
```ruby
    recoverable.errors[:base] << I18n.t("devise.failure.not_approved")
```

look up [errors[]](https://guides.rubyonrails.org/active_record_validations.html#errors) for info about errors for object

**Comment by thuy-econsys** [comment]
fixed by redirecting path after successful signup of unapproved user to `new_user_session_path` as default redirect goes to `root_path` which is not authorized for unapproved user which leads to 2nd redirect to sign-in page `new_user_session_path`, which explains the alert message "You need to sign in or sign up before continuing." for `en.devise.failure.unauthenticated`.

so going straight to `new_user_session_path` solves the problem. 

solved by [PR#14](https://github.com/thuy-econsys/rails_app/pull/14)
